android/app/src/test/java/org/smartregister/fhircore/fragment/PatientListFragmentTest.kt

```diff
@@ -174,14 +165,7 @@ class PatientListFragmentTest : FragmentRobolectricTest() {
       Pair(mutableListOf(patient), Pagination(totalItems = 1, pageSize = 5, currentPage = 1))
     )
     val container = getView<LinearLayout>(R.id.empty_list_message_container)
-    val buttonLayout =
```

Comment:
can you please add some new tests to cover as much as possible of `PatientListFragment`, this shows the uncovered lines, https://coveralls.io/builds/41754631/source?filename=src%2Fmain%2Fjava%2Forg%2Fsmartregister%2Ffhircore%2Ffragment%2FPatientListFragment.kt